packages/fim/manifest.yml

```
@@ -72,6 +72,16 @@ vars:
     required: false
     default:
       - sha1
+  - name: force_backend
+    type: text
+    title: File Event Source
+    description: |
+      Forces a particular event source for file events on Linux. `fsnotify` does not provide
+      user information. Supported types are:
+      `ebpf`, `fsnotify`, `kprobe`.
```

Suggest using a select type
```suggestion
    type: select
    title: File Event Source
    description: |
      Forces a particular event source for file events on Linux. `fsnotify` does not provide
      user information.
    options:
     - text: ebpf
       value: ebpf
     - text: fsnotify
       value: fsnotify
     - text: kprobe
       value: kprobe
```